src/rt/arc.rs

```diff
@@ -60,6 +64,10 @@ impl Arc {
         self.obj.branch(Action::RefInc);
 
         rt::execution(|execution| {
+            if execution.log {
+                println!("Arc::ref_inc");
```

Comment:
WDYT about including the ref count after increment in this message? We'd have to move the message down to where the ref count is incremented.

Reply:
Done.